### PR TITLE
CORDA-1410: Fix Jolokia runtime exception upon missing Jolokia jar file in driver…

### PR DESCRIPTION
**Pull request by josecoll**
…s directory.

Minor code refactoring/clean-up.
Updated old 3.0-SNAPSHOT references in sample tests and plugins.

### CODE REVIEW

File: cordformation/src/test/kotlin/net/corda/plugins/CordformTest.kt [hunk 1]
```diff
@@ -17,7 +17,7 @@ class CordformTest {
     private lateinit var buildFile: File
 
     private companion object {
-        const val cordaFinanceJarName = "corda-finance-3.0-SNAPSHOT"
+        const val cordaFinanceJarName = "corda-finance-3.1-corda"
```
Comment (chrisr3) [inline cordformation/src/test/kotlin/net/corda/plugins/CordformTest.kt]: This isn't a great place for the version number - isn't this dependent on `$project.version`?

File: cordformation/src/test/resources/net/corda/plugins/DeploySingleNodeWithCordapp.gradle [hunk 2]
```diff
@@ -1,7 +1,7 @@
 buildscript {
     ext {
         corda_group = 'net.corda'
-        corda_release_version = '3.0-SNAPSHOT' // TODO: Set to 3.0.0 when Corda 3 is released
+        corda_release_version = '3.1-corda'
```
Comment (chrisr3) [inline cordformation/src/test/resources/net/corda/plugins/DeploySingleNodeWithCordapp.gradle]: Do these Gradle tests actually work yet? These `build.gradle` files are test resources, and so we can expand properties via a `processTestResources {}` block.

**Comment by josecoll** [comment]
@chrisr3 CordForm tests do not seem to work before or after updating the corda version references. I will back out my changes related to this as they are not relevant to the code fix.

**Comment by josecoll** [comment]
@Clintonio any idea why the CordFormTest do not currently work?
Also worth noting that there are numerous references to Jolokia 1.3.7 (the version was recently updated in Corda OS to 1.5.0).

**Pull request merged**